app/views/assessments/_edit_problems.html.erb

```diff
@@ -1,7 +1,35 @@
-<script type="text/javascript">
-  is_positive_grading = <%= @is_positive_grading %>;
-  has_annotations = <%= @has_annotations %>;
-</script>
+<% content_for :javascripts do %>
+    <script type="text/javascript">
```

Comment:
General style nit w.r.t. javascript code: prefer using `let/const` instead of `var`